auth/client/src/main/resources/schema/elytron-client-1_4.xsd

```diff
@@ -346,9 +423,39 @@
                     </xsd:documentation>
                 </xsd:annotation>
             </xsd:element>
+            <xsd:element name="openssh-private-key" type="openssh-private-key-type">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        An encoded private key in OpenSSH format. Only the private key is needed to generate a KeyPair with
+                        the public and private keys.
+                    </xsd:documentation>
+                </xsd:annotation>
+            </xsd:element>
         </xsd:all>
     </xsd:complexType>
 
+    <xsd:complexType name="openssh-private-key-type">
+    <xsd:annotation>
+        <xsd:documentation>
+            An encoded private key in OpenSSH format. Only the private key is needed to generate a KeyPair with
+            the public and private keys.
+        </xsd:documentation>
+    </xsd:annotation>
+        <!-- The Passphrase used to encode the private key if applicable -->
+        <xsd:choice minOccurs="0" maxOccurs="1">
+            <xsd:element name="credential-store-reference" type="credential-store-reference-type"/>
+            <xsd:element name="clear-passphrase" type="clear-password-type"/>
+            <xsd:element name="masked-passphrase" type="masked-password-type"/>
+        </xsd:choice>
+        <xsd:attribute name="openssh-private-key" type="xsd:string" use="required">
```

Comment:
Since the element is already called `openssh-private-key` maybe this attribute name could be simplified to just `pem` or something like that?